pyproject.toml

```diff
@@ -1,8 +1,9 @@
 [tool.poetry]
 name = "wattile"
 version = "0.1.0"
-description = ""
+description = "Deep learning algorithms for probabilistic forecasting of time series building performance data."
```

Comment:
let's use "Probabilistic Deep Learning-based Forecasting of Building Energy Consumption"

Reply:
yes please!